packages/jsii-pacmak/lib/targets/python.ts

```diff
@@ -1429,6 +1429,13 @@ class PythonModule implements PythonType {
       code.line(`from ${'.'.repeat(distanceFromRoot + 1)}_jsii import *`);
 
       this.emitRequiredImports(code, context);
+      if (this.assembly.bin) {
```

Comment:
I like being explicit here (JS' truthiness/falsiness is a little awkward)

```suggestion
      if (this.assembly.bin != null) {
```